etna/transforms/timestamp/holiday.py

```diff
@@ -48,7 +70,7 @@ def _fit(self, df: pd.DataFrame) -> "HolidayTransform":
 
     def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         """
-        Transform data from df with HolidayTransform and generate a column of holidays flags.
+        Transform data from df with HolidayTransform and generate a column of holidays flags or its titles (NO_HOLIDAY if none).
```

Comment:
I think this detail about `NO_HOLIDAY` should be in description of the class at the very top for users to see it. It should be part of [extended summary](https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary).